doc/visual-programming/source/widgets/evaluate/calibrationplot.md

```diff
@@ -7,21 +7,41 @@ Shows the match between classifiers' probability predictions and actual class pr
-4. Press *Save Image* if you want to save the created image to your computer in a .svg or .png format.
-5. Produce a report.
+   - *Show rug*: If enabled, ticks are displayed at the bottom and the top of the graph, which represent negative and positive examples respectively. Their position corresponds to the classifier's probability prediction and the color shows the classifier. At the bottom of the graph, the points to the left are those which are (correctly) assigned a low probability of the target class, and those to the right are incorrectly assigned high probabilities. At the top of the graph, the instances to the right are correctly assigned high probabilities and vice versa.
+   - Curves for individual folds: a curve is displayed for each fold from the [Test and Score](testandscore.md) widget.
+2. Choose which classifiers to plot. The diagonal represents optimal behavior when *Calibration curve* is selected; the closer the classifier's curve gets, the more accurate its prediction probabilities are.
+3. Select the metric to calibrate:
+   - *calibration curve*: displays calibration curves for multiple models. It outputs a model with smoothed probabilities (converting them). The options for smoothing functions are [Sigmoid](https://en.wikipedia.org/wiki/Sigmoid_function) or [Isotonic](https://en.wikipedia.org/wiki/Isotonic_regression) function.
+   - *classification accuracy*: displays classification accurracy at changing probability thresholds. Threshold can be set by dragging the vertical line left or right.
+   - *F1*: displays F1 score at changing probability thresholds. Threshold can be set by dragging the vertical line left or right.
+   - *sensitivity and specificity*: displays the relationship between [sensitivity and specificity](https://en.wikipedia.org/wiki/Sensitivity_and_specificity) at changing probability thresholds. Threshold can be set by dragging the vertical line left or right.
+   - *precision and recall*: displays the relationship between [precision and recall](https://en.wikipedia.org/wiki/Precision_and_recall) at changing probability thresholds. Threshold can be set by dragging the vertical line left or right.
+   - *pos and neg predictive value*: displays the relationship between [positive and negative predictive values](https://en.wikipedia.org/wiki/Positive_and_negative_predictive_values) at changing probability thresholds. Threshold can be set by dragging the vertical line left or right.
+   - *true and false positive rate*: displays the relationship between [TP and FP rate](https://en.wikipedia.org/wiki/False_positives_and_false_negatives) at changing probability thresholds. Threshold can be set by dragging the vertical line left or right.
+4. If *Apply Automatically* is ticked, changes are communicated automatically. Alternatively, click *Apply*.
+
```

Comment:
After removing the repeated sentence from the above, I'd add some text about outputs:

When the widget shows the calibration curve, it outputs a calibrated model, whose predicted probabilities are tuned to better match the actual probabilities.

When showing other curves, the widget outputs a model that gives the same probabilities, but class predictions are made at thresholds different from 0.5. For instance, if we drag the vertical line, which is shown in the graph, to 0.3, the widget will output a model that will predict a positive class if its probability exceeds 30 %.

The widget cannot output a model if the input data contains models obtained from multiple runs (for instance from cross validation or repeated sampling). If there are multiple input models obtained by different algorithms, only a single one must be chosen in order to have it on the output. The widget also cannot output calibrated model for non-binary classes.